ord_schema/visualization/generate_text_test.py

```diff
@@ -37,8 +37,8 @@ def setUp(self):
             ))
         reaction.inputs['dummy_input'].components.add().CopyFrom(
             message_helpers.build_compound(
-                name='C1OCCC1',
-                smiles='THF',
+                name='THF',
```

Comment:
oops

Reply:
Yeah, I'm not quite sure why the test was passing before.